dist_tables/ddl.rst

```diff
@@ -61,6 +61,30 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
+------------------
+
+Table rows are stored in shards across a Citus cluster. When all rows needed by a query are available in a single shard then the tables named by the query are said to be *co-located for the query*. (Tables which are co-located for the majority of queries in an application are simply called co-located tables.) Queries with co-located tables excute faster because Citus doesn't need to shuffle data over the network.
+
+When the database administrator changes the number of worker nodes in a cluster, they run the Citus shard rebalancer to redistribute shards evenly. It's important to preserve table co-location during this process to
```

Comment:
Could we remove this, and the below shard-rebalancer discussion. While useful information, it probably doesn't need to be front-and-center for this section. Plus, since shard-rebalancer is enterprise only, it limits its usefulness here.